### PR TITLE
Update maestrano-connector rails to 1.0.2

### PR DESCRIPTION
**Pull request by x4d3**

### CODE REVIEW

File: app/models/entities/invoice.rb [hunk 1]
```diff
@@ -34,10 +34,7 @@ def self.can_update_external?
   end
 
   def self.references
-    [
-        {reference_class: Entities::SalesOrder, connec_field: 'sales_order_id', external_field: 'order_id'},
-        {reference_class: Entities::Person, connec_field: 'person_id', external_field: 'customer/id'}
-    ]
+    %w(sales_order_id person_id lines.item_id)
```
Comment (Berardpi) [inline app/models/entities/invoice.rb]: lines/item_id

File: config/settings/development.local.yml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+environment: 'local'
```
Comment (Berardpi) [inline config/settings/development.local.yml]: This file should not be commited

File: config/settings/development.yml [hunk 3]
```diff
@@ -0,0 +1,6 @@
+environment: 'local'
+app_host: 'http://localhost:3001/'
+
+default:
+  api_host: 'https://maestrano.com'
+  connec_host: 'http://localhost:8080'
```
Comment (Berardpi) [inline config/settings/development.yml]: Could you put Connec! prod instead here?

**Comment by Berardpi** [comment]
You've subscribed to connec! webhooks on invoices, do you really need them as from what I understood they are not sent to Shopify?

**Comment by x4d3** [comment]
@Berardpi yes, you are right. I've corrected

**Pull request merged**